flume-ng-channels/flume-kafka-channel/src/main/java/org/apache/flume/channel/kafka/KafkaChannel.java

```diff
@@ -269,7 +296,12 @@ private void setProducerProps(Context ctx, String bootStrapServers) {
     // Defaults overridden based on config
     producerProps.putAll(ctx.getSubProperties(KAFKA_PRODUCER_PREFIX));
     producerProps.put(ProducerConfig.BOOTSTRAP_SERVERS_CONFIG, bootStrapServers);
-
+    //  The default value of `ssl.endpoint.identification.algorithm` is changed to `https`, since kafka client 2.0+
+    //  And because flume does not accept an empty string as property value,
+    //  so we need to use alternative custom property `ssl.disableTLSHostnameVerification` to check if enable fqdn check.
```

Comment:
This line also greater than 120 characters

Reply:
fixed

Reply:
@tmgstevens please review again. 😁